README.md

```diff
@@ -0,0 +1,30 @@
+Documentation
+=============
+
+This repository provides a script which call two playbooks where a VM,
+previously created, is configured with all the packages and configuration
+required to install omnibus and build codethink-toolchain with omnibus
+(https://github.com/CodethinkLabs/omnibus-codethink-toolchain).
+
+This ansible script assume the CentOS VM is accessible by ssh client using
```

Comment:
I think it would be helpful for people who are not involved to specify which version of CentOS we used for creating the Ansible script.